src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/ValidForFlag.cs

```diff
@@ -21,6 +21,7 @@ public enum ValidForFlag
         Sql2017 = 0x40,
         SqlDw = 0x80,
         SqlOnDemand = 0x100,
+        AzureSqlDWGen3 = 0x200,
```

Comment:
You don't want to add this to AllAzure or All?

Although it looks like you're not using this at all - you just use the SqlDw flag instead. Is that intentional?

Reply:
@Charles-Gagnon I'll try adding to AllAzure and All, it makes sense to be there.  I reuse the SqlDW flag since that seemed to have the correct set of SMO properties that worked with the test server I have.  The other flags were trying to read properties that hadn't been set.

Reply:
What's the difference between SqlDw and AzureSqlDWGen3?

Reply:
@shueybubbles Gen3 supports DB pools and SqlDw does not.  I'll see if there is a way to get rid of this flag and only use SqlDw but there were some issues when I tried that previously while debugging.